test/ttmlir/Conversion/StableHLOToTTIR/concat_op.mlir

```diff
@@ -0,0 +1,13 @@
+// REQUIRES: stablehlo
+// RUN: ttmlir-opt --stablehlo-to-ttir-pipeline %s | FileCheck %s
+#any_device = #tt.operand_constraint<dram|l1|scalar|tile|any_device|any_device_tile>
+module @jit_concat attributes {} {
+  func.func public @test_concat(%input0: tensor<32x32xf32>, %input1: tensor<32x64xf32>) -> tensor<32x96xf32> {
```

Comment:
It is confusing to see `%input0` and `%input1` as function argument and then `%arg0` and `%arg1` in `CHECK` statement. I'll suggest to rename `%input0` and `%input1` to `%arg0` and `%arg1`.